spinnaker/provider.go

```diff
@@ -49,17 +51,30 @@ type gateConfig struct {
 func providerConfigureFunc(data *schema.ResourceData) (interface{}, error) {
 	server := data.Get("server").(string)
 	config := data.Get("config").(string)
-	ignore_cert_errors := data.Get("ignore_cert_errors").(bool)
+	ignoreCertErrors := data.Get("ignore_cert_errors").(bool)
+
+	var cfg spin_config.Config
+	var err error
+
+	if config != "" {
+		cfg, err = spin_config.ParseFromFile(config)
+	} else {
+		cfg, err = spin_config.Parse()
+	}
+
+	cfg.Gate.Insecure = ignoreCertErrors
```

Comment:
Should this maybe be an `||` to merge the config file and provider values? Or should the config file's insecure option be ignored completely?